_targets.R

```diff
@@ -22,7 +24,7 @@ list(
   # Read in data from gage-flow-conditions pipeline output
   tar_target(
     dv,
-    read_csv("https://labs.waterdata.usgs.gov/visualizations/data/flow_conditions_202204.csv", col_types = "cTnnnn")
+    read_csv("/Users/cnell/Documents/Projects/gage-conditions-gif/2_process/out/dv_stats_202205.csv", col_types = "cTnnnn")
```

Comment:
Is this set up as an RStudio project? Seems like this file path ought to be just `2_process/out/...'?